### PR TITLE
2023-06 release notes

### PR DESCRIPTION
**Pull request by fredvisser**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Release notes for 2023-06

### Why should this Pull Request be merged?

Standard release process

### What testing has been done?

Manual review

### CODE REVIEW

**Review by JD-Robertson — APPROVED** [review]

File: release-notes/2023-06/readme.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# SystemLink Enterprise release 2023-06 Release Notes
+
+The 2023-06 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the release year-release-month to the release year-release-month
+
+- 'assetservice >= 0.0.283'
+    - Description: New service was introduced in SystemLink chart. Any PVC that contains assetservice in its name needs to be removed prior to the upgrade. The following privileges are being used: asset:Query, asset:Create, asset:Update, asset:Delete.
```
Comment (MelissaHilliard) [inline release-notes/2023-06/readme.md:8]:
```suggestion
    - Description: This release introduced a new service in SystemLink chart. Before you upgrade, remove any PVC with assetservice in its name. The following privileges are being used: asset:Query, asset:Create, asset:Update, asset:Delete.
```
Is there a less passive way to mention the privileges? What is using them?

File: release-notes/2023-06/readme.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    - Installation guide: Remove pvc that have assetservice in its name.
```
Comment (MelissaHilliard) [inline release-notes/2023-06/readme.md:9]: PVCs?

**Review by MelissaHilliard — APPROVED** [review]

File: release-notes/2023-06/readme.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# SystemLink Enterprise release 2023-06 Release Notes
+
+The 2023-06 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
```
Comment (lnowotny) [inline release-notes/2023-06/readme.md:3]: This should probably just say '2023-06 release' and not mention 'bundle'.

**Review by lnowotny — APPROVED** [review]

File: release-notes/2023-06/readme.md (hunk 1 above)
Comment (fredvisser) [reply release-notes/2023-06/readme.md]: Reformatted it to:

```
- `assetservice 0.0.283`
    - New in 2023-06 release
    - This service uses the following privileges: asset:Query, asset:Create, asset:Update, asset:Delete
    - Before you upgrade, remove any PersistentVolumeClaims (PVCs) with `assetservice` in its name
 ```

File: release-notes/2023-06/readme.md (hunk 2 above)
Comment (fredvisser) [reply release-notes/2023-06/readme.md]: Defined it as "PersistentVolumeClaims (PVCs)" - 👍

**Pull request merged**